### PR TITLE
Very-minor tweaks to docs/tests

### PR DESCRIPTION
**Pull request by DougBurke**
# Summary

Fix a minor test issue and an unused reference in the documentation.

# Details

I noticed these problems during other unrelated work (from building CIAO and building the sphinx documentation). They are really-minor changes so I put them together so I wouldn't forget about them. The test issue is something I am sure I added, and the unused reference is from #1303 which removed the use of the reference but not the reference itself (rather than remove the reference I just added a reference to the website in the acknowledgements section as it seemed appropriate, and replaces the use of the web link in the text).

### CODE REVIEW

File: sherpa/sim/__init__.py [hunk 1]
```diff
@@ -30,9 +30,9 @@
 ----------------
 
 The original version of the code was developed by the CHASC
-Astro-Statistics collaboration http://hea-www.harvard.edu/AstroStat/,
-and was called pyBLoCXS. It has since been developed by the
-Chandra X-ray Center and weas added to Sherpa in version 4.5.1.
+Astro-Statistics collaboration, and was called pyBLoCXS [1]_. It has
+since been developed by the Chandra X-ray Center and weas added to
```
Comment (hamogu) [inline sherpa/sim/__init__.py:34]:
```suggestion
since been developed by the Chandra X-ray Center and was added to
```

Comment (DougBurke) [reply sherpa/sim/__init__.py]: At least this wasn't my typo.

Well, actually, I think it was my typo. But from old me, not this-PR me...

**Comment by DougBurke** [comment]
@hamogu - sorry, I added another simple fix I has added in #1352 but it doesn't need to be there and fits in here.

**Comment by DougBurke** [comment]
Hmm. The test failure is infrastructure related, not this code change

> Codecov: Uploader shasum does not match -- uploader hash: b09caed14d58bea0436487ae67d80d0bd743c64d96beac0ff35fe0336c504622  codecov, public hash: 4a19a16d3e3ec6966b3499bb166e08fa2226e64acc68edd97060631b01cf3ee3  codecov

I'll see if I can re-run them or this is related to AWS failures my twitterz is complaining about...

**Comment by DougBurke** [comment]
Ahh. The tests are back to green.

**Review by hamogu — APPROVED** [review]
I apologize for the delay. I know I looked at this but for some reason apparently did not submit the review.

I don't want to hold this up, but in the spirit of #1306 I'm tempted to ask for real-world examples that justify an isolated change of `==` to `is` without a full review of the code base to see if there are any other instances that this PR does not address ;-)

**Comment by DougBurke** [comment]
Well, I fixed the **only** case of  'please change is to ==' that python complained about. I'm not sure how else to identify other places where this could happen. I also think an isolated change like this is vastly different from the #1306 case where we **know** that there are many other problems.

**Pull request merged**